### DISCUSSION TITLE
Can we draw points using Annotorious 3.x?

### DISCUSSION THREAD

**Discussion by Songhao-LI** [annotorious/annotorious#475] [Q&A] [answered]
Hi all,

How can we draw points using Annotorious 3.x? Is it possible to draw points by using the polygon type but only passing in one coordinate? Also, how can we define custom shapes by ourselves?

Thanks in advance!!

**Comment by rsimon** [comment, marked as answer]
Not at the moment. Please see here: https://github.com/orgs/annotorious/discussions/466
